File: C/beginner/1006.c
/*Including Libraries*/
#include <stdio.h>

/*Main Function*/

int main() {
    /*Variable declaration and initialization*/
    float a, b, c, media;

    /*Requesting values to user*/
    scanf("%f", &a);
    scanf("%f", &b);
    scanf("%f", &c);

    /*Doing the calculations*/
    media = ((a * 2) + (b * 3) + (c * 5)) / 10;

    /*Showing results*/
    printf("MEDIA = %.1f\n", media);

    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/

File: Haskell/beginner/1006.hs
-- Doing imports
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    a <- readLn :: IO Float
    b <- readLn :: IO Float
    c <- readLn :: IO Float
    
    -- Doing the calculations
    let media = ((a * 2) + (b * 3) + (c * 5)) / 10

    -- Showing results
    printf "MEDIA = %.1f\n" media

-- By Fernando Serra
-- https://github.com/fernandosserra